docker/Dockerfile

```diff
@@ -123,7 +123,7 @@ EXPOSE 9200 9300 9600 9650
 
 LABEL org.label-schema.schema-version="1.0" \
   org.label-schema.name="opensearch" \
-  org.label-schema.version="1.3.3" \
```

Comment:
Did we miss this last time and left it as `1.3.3` instead of  `1.3.4`

Reply:
Hey @kkhatua looks like yes, hence I have updated now.
Thanks